### PR TITLE
RIFEX-39 RNS pending domain detail

### PR DESCRIPTION
**Pull request by JONAF2103**
This PR adds the support for pending register details to allow the user to get back to a register process in progress. Here is a demo:

demo-13-05-2020-jonathan-pending-domain-details

* Adding the control to the register page to load pending register process.
* Adding register page to the details page on pending status
* Adding logic to the background to update the domain status when it's done.

This affects ui and service layers.

### CODE REVIEW

**Review by marcosmartinez7 — APPROVED** [review]
Lgtm in general, good job

File: app/scripts/controllers/rif/rns/index.js [hunk 1]
```diff
@@ -71,6 +72,11 @@ export default class RnsManager {
       address: this.address,
       store: this.store,
     });
+    this.container = new RnsContainer({
```
Comment (rorolopetegui) [inline app/scripts/controllers/rif/rns/index.js:75]: I like this!

File: ui/app/rif/components/searchDomains.js [hunk 2]
```diff
@@ -14,21 +14,35 @@ class SearchDomains extends Component {
...
+    })
+    }
+
   _handleKeyDown = (e) => {
     if (e.key === 'Enter') {
       const typedDomain = cleanDomainName(e.target.value);
-      // There is a limitation in manager that domains with less 5 characters are blocked
-      if (typedDomain.length < 5) {
+      // There is a limitation in rns that domains with less 5 characters are blocked
+      if (typedDomain.length <= 5) {
```
Comment (rorolopetegui) [inline ui/app/rif/components/searchDomains.js:36]: Please, delete the equals from here, typedDomain can be of 5 chars

File: ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js [hunk 3]
```diff
@@ -235,23 +259,25 @@ class DomainsDetailActiveScreen extends Component {
 }
 
 function mapStateToProps (state) {
-	const data = state.appState.currentView.params;
+	const params = state.appState.currentView.params;
```
Comment (rorolopetegui) [inline ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js:262]: Good refactor here!

File: ui/app/rif/pages/rns/register/index.js [hunk 4]
```diff
@@ -258,43 +269,40 @@ class DomainRegisterScreen extends Component {
         </div>
       ),
     };
-    if (currentStep === 'registering' && this.props.registeringProgress <= 120) {
+    if (currentStep === 'waitingForRegister') {
```
Comment (rorolopetegui) [inline ui/app/rif/pages/rns/register/index.js:272]: Maybe we can move this strings to constants, or its irrelevant?

**Review by rorolopetegui — CHANGES_REQUESTED** [review]
Some minor changes required, but it lgtm

File: ui/app/rif/components/searchDomains.js (hunk 2 above)
Comment (JONAF2103) [reply ui/app/rif/components/searchDomains.js:36]: Oh ok this was a merge issue i guess.

File: ui/app/rif/pages/rns/register/index.js (hunk 4 above)
Comment (JONAF2103) [reply ui/app/rif/pages/rns/register/index.js:272]: It's internal to the component, i mean it's for internal management of the component so i guess is not necessary

**Pull request merged**